Dockerfile

```diff
@@ -2,5 +2,28 @@ FROM jangrewe/gitlab-ci-android
 
 RUN apt-get update && \
   apt-get install -y --no-install-recommends \
+  nodejs \
   build-essential \
   && rm -rf /var/lib/apt/lists/*
+
+ENV PATH="$PATH:$ANDROID_HOME/tools:$ANDROID_HOME/platform-tools:$ANDROID_HOME/tools/bin"
+
+RUN echo $PATH
+
+WORKDIR "/leanplum"
+
+CMD bash
+
+RUN rm -f local.properties
+
+RUN sdkmanager --list
+
+RUN sdkmanager --no_https --licenses
+
+RUN sdkmanager --no_https emulator tools platform-tools "platforms;android-26" "system-images;android-26;google_apis;x86" --verbose
+RUN echo "Creating an Emulator"
+RUN echo no | avdmanager create avd -n "x86" --package "system-images;android-26;google_apis;x86" --tag google_apis
```

Comment:
This command and the sdkmanager command both depend on which package and architecture you use. With this in mind, it would behoove us to:

1. Extract names of things like "system-images;android-26;google_apis;x86" to variables.
2. Do the sdkmanager and avdmanager commands in the same `RUN` (like you see in `apt-get`) so they are bundled in the same docker layer.